models/recharge__charge_line_item_history.sql

```diff
@@ -28,7 +28,7 @@ with charges as (
         'discount' as line_item_type
     from charge_discount_codes
     left join discounts
-        using(discount_id)
+        using(discount_id) 
```

Comment:
We found a while back that the `using` function has some gaps in other warehouses. Would you be able to adjust any instances of using in this package to be left joins